<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>522</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>xml生成器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QListView" name="listFiles"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>清空</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="editPath">
           <property name="placeholderText">
            <string>目标路径</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnPath">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="oldRate">
           <property name="text">
            <string>原始倍率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioOldRate_1_0">
           <property name="text">
            <string>1.0x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioOldRate_1_5">
           <property name="text">
            <string>1.5x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioOldRate_2_0">
           <property name="text">
            <string>2.0x</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="newRate">
           <property name="text">
            <string>目标倍率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioNewRate_1_0">
           <property name="text">
            <string>1.0x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioNewRate_1_5">
           <property name="text">
            <string>1.5x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioNewRate_2_0">
           <property name="text">
            <string>2.0x</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="btnConvert">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icon/go.png</normaloff>:/icon/go.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textLog">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
